### PR TITLE
update customer model

### PR DESCRIPTION
**Pull request by cco-franchine**
## [0.3.1] - 2023-10-02

- Updated `customer` model  to include customer number and taxonomy details

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SEMANTIC_VER=0.2.1+
+SEMANTIC_VER=0.3.1+
```
Comment (cco-joshua) [inline Makefile:1]: Would you consider a revision change to 0.2.2... since the addition of a field to the customer is not backwards breaking?

**Review by cco-joshua — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (cco-franchine) [reply Makefile:1]: yes!

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,15 @@ All notable changes in io-sdk-golang will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
 ## [0.3.0] - 2023-08-21
 
 - Modified query options encoding for optimal use with CCO microservices
 
+## [0.2.2] - 2023-10-02
```
Comment (sunflailOOH) [inline CHANGELOG.md:13]: Looks like this one needs to be above the above one and with a later version?

Comment (cco-joshua) [reply CHANGELOG.md:13]: omfg... there is a v0.3.0, but the Makefile wasn't updated. I think this is my bad.

**Review by sunflailOOH — APPROVED** [review]
Looks good after the change log and semantic change in comments

**Review by cco-robin — APPROVED** [review]
looks good other than the versioning kelson pointed out

File: Makefile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-SEMANTIC_VER=0.2.1+
+SEMANTIC_VER=0.2.2+
```
Comment (cco-joshua) [inline Makefile:1]: Sorry, scratch my last comment... this should be v0.3.1, and the comment in CHANGELOG.md to match. My apologies for the round-about.

**Pull request merged**